### DISCUSSION TITLE
[useCannon] - Is there any way to Pause/UnPause the Physics world ?

### DISCUSSION THREAD

**Discussion by htgmanics** [pmndrs/use-cannon#191] [Q&A]
Hi,
I'm using useCannon for my project.
I'd like to pause the physics world when the mouse leave the canvas,
and resume it when the i hover the canvas again.

Is it possible to do so ? Many Thanks!

**Comment by fgarrec0397** [comment]
Yes, on your `<Physic>` component there is a props called `isPaused`
